client/my-sites/checkout/checkout-thank-you/jetpack-thank-you-card.jsx

```diff
@@ -642,7 +642,7 @@ class JetpackThankYouCard extends Component {
 					<a
 						className={ classNames( 'button', 'thank-you-card__button', { 'is-placeholder': ! buttonUrl } ) }
 						href={ buttonUrl }>
-						{ translate( 'Visit Your Site' ) }
+						{ translate( 'Visit your site' ) }
```

Comment:
:disappointed: 38 existing translations will be lost with this change.

Reply:
I know :(